src/herder/TransactionQueue.cpp

```diff
@@ -396,8 +398,8 @@ TransactionQueue::canAdd(TransactionFrameBasePtr tx,
     int64_t totalFees = feeStateIter == mAccountStates.end()
                             ? 0
                             : feeStateIter->second.mTotalFees;
-    if (getAvailableBalance(ltx.loadHeader(), feeSource) - newFullFee <
-        totalFees)
+    if (getAvailableBalance(ltx.loadHeader(), feeSource) <
+        totalFees + newFullFee)
```

Comment:
So we actually don't get to this line, because we fail earlier in `checkValid` due to account insufficient balance to cover `newFullFee`. To get here, we either need an int64_max balance, or a bunch of txs in the queue that add up to int64_max. I'll just keep this logic like in master (without moving newFullFee to RHS) for now